CONTRIBUTING.adoc

```diff
@@ -31,6 +31,7 @@ Then you can just replace the Plugin POM version in your downstream PRs by this
 ### Managing release notes
 
 * Changelog drafts are automatically by link:https://github.com/toolmantim/release-drafter[Release Drafter]
+** See the documentation about Jenkins Release Drafter configuration link:https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.adoc[here]
```

Comment:
@oleg why did you restore this it is garbage URL.

Reply:
I fixed it